2-alcance/README.md

```diff
@@ -1,3 +1,286 @@
+
+##### Refugio
+
+Marta lleva a España y busca a un alojamiento en el País Vasco, debe hacer clic en la pestaña del refugio encontrar los diferentes servicios que puede ayudarle. 
+
+##### Alimentación 
+
+Marta es en el País Vasco y le necesita buscar comida, en este pestana, puede encontrar las diferentes lugares y las bancos alimentos. 
+
+Si haces clic en educacion te llevara a una pagina donde estará la información de los centros de educación en el País Vasco y donde pueden ingresar los niños, jóvenes o adultos. Por otra parte también se indicará la documentación requerida.
+
+##### Género:
+
+Clicando en la pestaña Género tendrá acceso a la información de centros o instituciones que trabajan en pro de la protección a las mujeres y niños que son vulnerables y probablemente abusados o maltratados en el País Vasco. 
+
+##### Empleo:
+
+Si Juan llega al País Vasco en busca de mejores oportunidades y brindarle a su familia una estabilidad económica, deberá clicar en la pestaña empleo y encontrará las ofertas que se ajusten a sus necesidades. 
+
+#### Foro (Estelle)
+
+Marta tiene muchas preguntas y quiere conocer las opiniones de los otros inmigrantes entonces clica en “Foro” arriba a la derecha. De esta manera puede discutir de lo que quiera con la comunidad de Aupe.
+
+#### Contacto (Estelle)
+
+Ciclando en “Contacto” Marta quiere contactarnos porque tiene una pregunta sobre una de los entidades asociada. En la página va a llenar un formulario de contacto con su nombre, su apellido y su correo electrónico, para hacer su pregunta. 
+
+### Buscador (Estelle)
+
+Marta querría conocer los trámites a seguir para hacer una demanda de asilo pero no se recuerda como encontrarlo, así clica en la lupa del buscador arriba a la derecha y taclea “demanda de asilo” y hacer “enter”, está en la página donde hay todas las informaciones para demandar el asilo. 
+
+### Las idiomas (Estelle)
+
+Marta viene de Venezuela y por supuesto habla espanol pero su novio esta mas comodo con el ingles porque viene de América del Norte. Marta va a clicar en la pestaña “Idiomas” para cambiar la lengua y cuando el menú se abre clica en “Inglés”, el sitio se actualiza automáticamente. 
+
+## Requerimientos de contenido 
+
+### CONTENIDO
+
+#### Audiencia:
+
+El público objetivo de Aupa se estableció mediante un análisis del origen, la cantidad de migrantes en el País Vasco y la edad promedio. A raíz de ello y del estudio de “personas” en la etapa previa, consideramos encaminar nuestro sitio web a las necesidades que Martha Morales migrante Nicaragüense nos manifestó. Teniendo en cuenta que: 
+
+- Su frustración es estar indocumentada.
+
+- Sus objetivos son conseguir la documentación, tener sus estudios homologados y un lugar donde pueda estar cómoda con su familia.
+
+#### Mensaje: 
```

Comment:
Aquí hacéis una descripción de las **secciones**, cuando lo que corresponde primero es que hagáis un listado y descripción de los **tipos de contenidos** que vais a tener. Mencionáis noticias, videos, páginas con enlaces a trámites en sitios externos, etc., de manera bastante desorganizada.